Sources/MapboxDirections/DirectionsOptions.swift

```diff
@@ -129,6 +129,8 @@ open class DirectionsOptions: Codable {
         self.waypoints = waypoints
         self.profileIdentifier = profileIdentifier ?? .automobile
         
+        if waypoints.count < 2 { assertionFailure("waypoints array requires at least 2 waypoints") }
```

Comment:
This doesn't actually fix the crash. The assertion will be visible only in the debugger, but the app will still crash in production.